sdk/vault-client/tsconfig.test.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "noEmit": true
+    "noEmit": true,
```

Comment:
Is that ok with tsconfig? it breaks package.json iirc

Reply:
tsconfig allow comments and trailing commas